etc/test.bash

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+ROOT_DIR="$DIR/../"
+cd $ROOT_DIR
+
+if [[ "$1" == "cover" ]]; then
+	COVER=-coverprofile=coverage.txt
+fi
+
+# race isn't supported on the pi or jetson (and possibly other arm boards)
+# https://github.com/golang/go/issues/29948
+if [ "$(uname -m)" != "aarch64" ] || [ "$(uname)" != "Linux" ]; then
+	RACE=-race
+fi
+
+gotestsum --format standard-verbose --jsonfile json.log -- -tags=no_skip $RACE $COVER ./...
```

Comment:
our scheme of grabbing the pid, trapping, and all that is not needed since this is one command now. We should be able to reduce this to the gotestsum command here followed by the analyzers